go/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/keep-network/tbtc/go
+
+go 1.13
+
+require (
+	github.com/ethereum/go-ethereum v1.9.10
+	github.com/ipfs/go-log v1.0.4
+	github.com/keep-network/keep-common v1.2.1-0.20201015091820-496b0c87d6f5
+	github.com/keep-network/keep-core v1.3.0
```

Comment:
If we don't generate commands, can we avoid having a dependency to `keep-core`? 

It looks... wrong. Although commands would be a nice addition we don't need them rigtht now.

Reply:
We can't disable command generation without changing the generator code in `keep-common`.

Reply:
We can try to update the generator to allow to ship a parameter to skip command generation. It feels better to me than requiring `tbtc` to depend on `keep-core`.

Reply:
Ok, but can we do it separately? It's a non-trivial change and I don't want to block here now.

Reply:
We can always extract bindings to a separate PR - contracts are not changing so we don't need to regenerate them daily - and do generation here, once we solve the problem with commands. It is really hard to anticipate what is a consequence of adding `keep-core` dependency from `tbtc` to our deployment lifecycle, even if it is temporary.

Worth noting, maybe it's not about disabling command generation but about moving some `keep-core` code out to `keep-common`?

Reply:
Ok, I have done the change about optional commands.